src/lae_site/handlers/create_subscription.py

```diff
@@ -59,16 +54,13 @@ def __init__(self, get_signup, mailer, stripe):
         self._get_signup = get_signup
         self._mailer = mailer
         self._stripe = stripe
-
+        self._cross_domain = cross_domain
+    
+    def addHeaders(self, request, cross_domain):
+      request.setHeader('Access-Control-Allow-Origin', cross_domain)
```

Comment:
Maybe a brief comment or docstring here about why we need Access-Control-Allow-Origin to be set on these responses.